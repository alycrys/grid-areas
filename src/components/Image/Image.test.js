import React from "react";
import renderer from "react-test-renderer";
import Image from "./Image";

it("renders correctly", () => {
  const wrapper = renderer.create(<Image src="source" alt="alt text" />);

  expect(wrapper.toJSON()).toMatchSnapshot();
});
