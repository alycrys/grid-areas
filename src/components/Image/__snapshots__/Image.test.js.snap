// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="alt text"
  src="source"
/>
`;
